src/ID3Helpers.js

```diff
@@ -96,55 +96,61 @@ module.exports.getTagsFromBuffer = function(filebuffer, options) {
     return getTagsFromFrames(frames, ID3Version, options)
 }
 
-function getFramesFromID3Body(ID3FrameBody, ID3Version, options = {}) {
+function isFrameDiscardedByOptions(frameIdentifier, options) {
+    if(options.exclude instanceof Array && options.exclude.includes(frameIdentifier)) {
+        return true
+    }
+
+    return options.include instanceof Array && !options.include.includes(frameIdentifier)
+}
+
+function getFramesFromID3Body(ID3TagBody, ID3Version, options = {}) {
     let currentPosition = 0
     const frames = []
-    if(!ID3FrameBody || !(ID3FrameBody instanceof Buffer)) {
+    if(!ID3TagBody || !(ID3TagBody instanceof Buffer)) {
         return frames
     }
 
-    let identifierSize = 4
-    let textframeHeaderSize = 10
-    if(ID3Version === 2) {
-        identifierSize = 3
-        textframeHeaderSize = 6
-    }
+    const frameIdentifierSize = (ID3Version === 2) ? 3 : 4
+    const frameHeaderSize = (ID3Version === 2) ? 6 : 10
 
-    while(currentPosition < ID3FrameBody.length && ID3FrameBody[currentPosition] !== 0x00) {
-        const bodyFrameHeader = Buffer.alloc(textframeHeaderSize)
-        ID3FrameBody.copy(bodyFrameHeader, 0, currentPosition)
+    while(currentPosition < ID3TagBody.length && ID3TagBody[currentPosition] !== 0x00) {
+        const frameHeader = Buffer.alloc(frameHeaderSize)
+        ID3TagBody.copy(frameHeader, 0, currentPosition)
```

Comment:
Do we need a copy? If we do read only operations, we could use a view instead with `Buffer.slice`.
https://nodejs.org/docs/latest-v10.x/api/buffer.html#buffer_buf_slice_start_end
```js
const frameHeader = ID3TagBody.slice(currentPosition)
```